packages/breadcrumbs/src/spectrum-breadcrumbs-item.css

```diff
@@ -0,0 +1,235 @@
+    );
+    opacity: 1;
+    color: var(
+        --highcontrast-breadcrumbs-separator-color,
+        var(
+            --mod-breadcrumbs-separator-color,
+            var(--spectrum-breadcrumbs-separator-color)
+        )
+    );
+    position: relative;
+}
+
+.separator:dir(rtl),
+:host([dir='rtl']) .separator {
+    transform: scaleX(-1);
+}
+
+:host {
+    box-sizing: border-box;
+    white-space: nowrap;
+    font-family: var(
+        --mod-breadcrumbs-font-family,
+        var(--spectrum-breadcrumbs-font-family)
+    );
+    font-size: var(
+        --mod-breadcrumbs-font-size,
+        var(--spectrum-breadcrumbs-font-size)
+    );
+    font-weight: var(
+        --mod-breadcrumbs-font-weight,
+        var(--spectrum-breadcrumbs-font-weight)
+    );
+    line-height: var(
+        --mod-breadcrumbs-line-height,
+        var(--spectrum-breadcrumbs-line-height)
+    );
+    display: inline-flex;
+    position: relative;
+}
+
+::slotted(sp-action-menu) {
+    margin-inline: var(
+        --mod-breadcrumbs-action-button-spacing-inline,
+        var(--spectrum-breadcrumbs-action-button-spacing-inline)
+    );
+    margin-block: var(
+        --mod-breadcrumbs-action-button-spacing-block,
+        var(--spectrum-breadcrumbs-action-button-spacing-block)
+    );
+    color: var(
+        --highcontrast-breadcrumbs-action-button-color,
+        var(
+            --mod-breadcrumbs-action-button-color,
+            var(--spectrum-breadcrumbs-action-button-color)
+        )
+    );
+}
+
+::slotted(sp-action-menu[disabled]) {
+    color: var(
+        --highcontrast-breadcrumbs-action-button-color-disabled,
+        var(
+            --mod-breadcrumbs-action-button-color-disabled,
+            var(--spectrum-breadcrumbs-action-button-color-disabled)
+        )
+    );
+}
+
+.item-link {
+    cursor: default;
+    box-sizing: border-box;
+    border-radius: var(
+        --mod-breadcrumbs-item-link-border-radius,
+        var(--spectrum-breadcrumbs-item-link-border-radius)
+    );
+    color: var(
+        --highcontrast-breadcrumbs-text-color,
+        var(
+            --mod-breadcrumbs-text-color,
+            var(--spectrum-breadcrumbs-text-color)
+        )
+    );
+    outline: none;
+    margin-block-start: var(
+        --mod-breadcrumbs-text-spacing-block-start,
+        var(--spectrum-breadcrumbs-text-spacing-block-start)
+    );
+    margin-block-end: var(
+        --mod-breadcrumbs-text-spacing-block-end,
+        var(--spectrum-breadcrumbs-text-spacing-block-end)
+    );
+    -webkit-text-decoration: none;
+    text-decoration: none;
+    display: block;
+    position: relative;
+}
+
+.item-link.is-disabled,
+:host([aria-disabled='true']) .item-link {
+    color: var(
+        --highcontrast-breadcrumbs-text-color-disabled,
+        var(
+            --mod-breadcrumbs-text-color-disabled,
+            var(--spectrum-breadcrumbs-text-color-disabled)
+        )
+    );
+}
+
+:host([href]) .item-link,
+:host([tabindex='0']) .item-link {
```

Comment:
This applies well when we use `sp-breadcrumb-item` with `href` attribute.
But with `value` attribute, this CSS will not apply, because there is not `tabindex="0"` on the `sp-breadcrumb-item` and it is not OK to add one there either.
This CSS should be on the `item-link` class itself, as it is on the Spectrum CSS.